geomstats/geometry/embedded_manifold.py

```diff
@@ -60,6 +117,7 @@ def extrinsic_to_intrinsic_coords(self, point_extrinsic):
         raise NotImplementedError(
             'extrinsic_to_intrinsic_coords is not implemented.')
 
```

Comment:
Do we want to add to_tangent as an abstractmethod too?

Reply:
It is already from the parent class `Manifold` :)